src/redux/User.redux/User.reducer.js

```diff
@@ -6,9 +6,11 @@ import { LOG_OUT, SET_USER } from '../actions';
  */
 const userInitialState = {
```

Comment:
let's leave this file alone for now. It's used for storing information on the currently logged in user, not the editing form